test_case/pdfMarkups.lsp

```diff
@@ -0,0 +1,162 @@
+; random sample file
+
+(defun sampleFunc (x y / a b c d)
```

Comment:
If the test case codes is from other open source project, please check its license.

Reply:
This isn't an issue, I pulled something out of my code base and it is completely non-functional without highly unique DLL's that I've made to perform PDF I/O and SharePoint operations from within AutoCAD.